contrib/gitlab/pkg/client/clientset/versioned/fake/clientset_generated.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by client-gen. DO NOT EDIT.
+
+package fake
```

Comment:
I'm just curious, since this is generated code, has it been generated by `go-client` directly, or have you used `operator-sdk` to generate it?

Reply:
It's generated by `go-client` directly